### ISSUE TITLE
Bug : v0.7 container failed to start

### ISSUE THREAD

**Issue by AdiAbuAli** [mrlt8/docker-wyze-bridge#149]
Hi,
I cant run latest 0.7 release due to can run docker API with attached path error.

Also seems NET_MODE doesnt accept LAN option or its my mobile browser :)
![Screenshot_20210929-055403_Samsung Internet](https://user-images.githubusercontent.com/89463853/135195568-09f472d9-af91-4309-9100-524d498efc5a.jpg)

**Comment by mrlt8** [comment]
Supervisord has been removed for home assistant compatibility.

Startup CMD now runs wyze_bridge.py on startup:
https://github.com/mrlt8/docker-wyze-bridge/blob/989fb5e8129299b10044ef73521501f192365cc8/app/Dockerfile#L33

**Comment by AdiAbuAli** [comment]
Yes it works now...I deleted the old container and lunched new one, instead of reset...
Thanks,

Thanks,
________________________________
